ehitchhiking-ui/src/app/shared/api-services/black-list-api.service.ts

```diff
@@ -0,0 +1,75 @@
+import {Injectable} from '@angular/core';
+import {User} from '@shared/models/user';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+import {ApiService} from '@shared/services/api.service';
+import {Car} from '@shared/models/car';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class BlackListApiService {
+	BLACKLIST_PASSENGERS: User[] = [];
+	BLACKLIST_DRIVERS: User[] = [];
+
+	constructor(private apiService: ApiService) {}
+	func1(data) {
+		this.BLACKLIST_PASSENGERS.splice(0, this.BLACKLIST_PASSENGERS.length);
+		data.forEach((obj) => {
+			this.BLACKLIST_PASSENGERS.push(
+				new User(obj.id, obj.firstName + ' ' + obj.lastName, '', obj.email, '', [
+					new Car('ferrari', 'pink', 'A3434B', 1),
+				])
+			);
+		});
+	}
+
+	func2(data) {
+		console.log(data);
+		while (this.BLACKLIST_DRIVERS.length > 0) {
+			this.BLACKLIST_DRIVERS.pop();
+		}
+		// console.log(this.BLACKLIST_DRIVERS.length);
+		data.forEach((obj) => {
+			this.BLACKLIST_DRIVERS.push(
+				new User(obj.id, obj.firstName + ' ' + obj.lastName, '', obj.email, '', [
+					new Car('ferrari', 'pink', 'A3434B', 1),
+				])
+			);
+		});
+		// console.log(this.BLACKLIST_DRIVERS.length);
+	}
+
+	doGetPassengers(curUser: User) {
```

Comment:
bad naming: doGetPassengers. rename to getBlockedPassengers

Reply:
renamed